src/lib/rted.rs

```diff
@@ -71,8 +71,9 @@ pub struct RTEDConfig<'a> {
     cost_w: Vec<Vec<usize>>,
     // T array from Demaine's algorithm, stores delta(Fv,Gij), v on heavy path.
     t: Vec<Vec<f64>>,
-    // Copy values from the `t` vector, in case in single compute period values
-    // are  overwritten before they are read due to a change of forest ordering.
+    // Copy values from the `t` vector. This is used when, in a single compute
+    // period, values are overwritten before they are read, due to a change of
+    // forest ordering.
```

Comment:
Note that the "compute period" refers to a function that hasn't been committed yet.